.github/workflows/jest.yml

```diff
@@ -0,0 +1,51 @@
+name: Jest
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+  schedule:
+    - cron: '24 21 * * 4'
```

Comment:
What is the cron job for?

Reply:
it was leftover from the github actions default job, this has been removed